containers/members/MemberStorageSelector.js

```diff
@@ -8,8 +8,10 @@ import humanSize from 'proton-shared/lib/helpers/humanSize';
 const FIVE_GIGA = 5 * GIGA;
 
 const MemberStorageSelector = ({ member, organization, onChange }) => {
-    const minPadding = 0;
-    const maxPadding = organization.MaxSpace - organization.AssignedSpace;
+    const step = GIGA;
+    const minPadding = Math.ceil(member.UsedSpace / GIGA) * GIGA;
+    const maxPadding =
+        Math.ceil((organization.MaxSpace - organization.UsedSpace + member.UsedSpace) / GIGA) * GIGA + step;
     const startNewMember = maxPadding > FIVE_GIGA ? FIVE_GIGA : maxPadding;
     const options = range(minPadding, maxPadding, GIGA).map((value) => ({ text: `${humanSize(value, 'GB')}`, value }));
```

Comment:
since `step = GIGA` has been defined, maybe the plan was to use it here as well?
`range(minPadding, maxPadding, step)`